Applications/Games/Overwatch/Online/script.js

```diff
@@ -10,19 +10,20 @@ var installerImplementation = {
             .name("Overwatch")
             .editor("Blizzard")
             .applicationHomepage("http://www.playoverwatch.com/")
-            .author("ImperatorS79")
+            .author("ImperatorS79, kreyren")
             .url("https://eu.battle.net/download/getInstaller?os=win&installer=Overwatch-Setup.exe")
         //The checksum is different each time you download
             .wineVersion(LATEST_STAGING_VERSION)
-            .wineDistribution("staging")
-            .wineArchitecture("amd64")
+            .wineDistribution("staging") // ESYNC required
+            .wineArchitecture("amd64") // Multilib preffered? (without setting wine architecture)
             .category("Games")
-            .executable("Overwatch.exe")
+            .executable("Battle.net.exe") // Needs verification
             .preInstall(function (wine/*, wizard*/) {
-                wine.windowsVersion("winxp");
+                wine.windowsVersion("win7");
                 wine.vcrun2015();
                 wine.corefonts();
                 wine.enableCSMT();
+                wine.DXVK(); // Relevant: https://github.com/doitsujin/dxvk/issues/846
```

Comment:
Please add 
```js
include(["engines", "wine", "verbs", "dxvk"]);
```
at the beginning of the file.

Reply:
Added, understood the concept of include for dependencies